javatests/arcs/android/host/AndroidEntityHandleManagerTest.kt

```diff
@@ -46,7 +46,7 @@ class AndroidEntityHandleManagerTest : LifecycleOwner {
 
     val entity1 = Person("Jason", 21.0, false)
     val entity2 = Person("Jason", 22.0, true)
-    lateinit var handleHolder: TestParticleHandles
+    lateinit var handleHolder: AbstractTestParticle.TestParticleHandles
```

Comment:
How about just calling it Handles? `AbstractTestParticle.Handles`

Reply:
Updated!